apps/omg_watcher/test/web/controllers/status_test.exs

```diff
@@ -49,4 +49,24 @@ defmodule OMG.Watcher.Web.Controller.StatusTest do
 
     started_apps |> Enum.each(fn app -> :ok = Application.stop(app) end)
   end
+
+  @tag fixtures: [:watcher_sandbox, :root_chain_contract_config]
+  test "status endpoint returns expected response format without worrying about the Content-Type header" do
+    response_body = TestHelper.rpc_call("status.get", nil, 200, "text/plain")
+    %{"version" => "1.0", "success" => true, "data" => data} = response_body
+
+    assert %{
```

Comment:
oh, actually I have a suggestion - let's have this test test only what it intends to test. Either just end the test on line 56 (with `assert`) or compare data from regular `TestHelper.success?("status.get")` call, instead of duplicating the detailed assertions